index.js

```diff
@@ -51,16 +54,10 @@ function renderCalendar() {
     for (let i = 1; i <= daysInMonth; i++) {
         let currentDate = new Date(SelectedMonth.getFullYear(), SelectedMonth.getMonth(), i);
 
-        let currentDayOfWeek = new Intl.DateTimeFormat("en-US", {
+        let currentDayOfWeek = new Intl.DateTimeFormat(undefined, {
             weekday: "short",
-        }).format(currentDate);
+        }).format(currentDate)[0].toUpperCase();
```

Comment:
This will probably make all the letters uppercase, I am not sure if that's what we are going for, but if we only want to make the first letter capital, we should add a css property to that class, e.g. `text-transform: capitalize;`